### ISSUE TITLE
Tooltip portal container issue

### ISSUE THREAD

**Issue by snownoop** [salesforce/design-system-react#2156]
I was trying to render **Tooltip** and was not able to set **portal container** for it to render in.
For **Modal** component we have something like **parentSelector**={} prop.

For **Tooltip** that is using **Dialog** component then we don't have any prop or ability to define portal container.

i found only **portalMount** prop on **Dialog**, but it is not passing to it from **Tooltip** component.

Can you advice is it possible **to render tooltip somewhere not to body**, and how ? If not can you enhance **Tooltip**, so it will pass **portalMount** to **Dialog** ?

**Comment by interactivellama** [comment]
You are correct, there’s not a way to make `overflowBoundaryElement` Tooltip not use the body right now. The functionality still needs to be added. I would be open to a `parentSelector` prop.

```
/**
	 * If `position:”overflowBoundaryElement”`, a portal will be created. This function’s return value is the mount node to insert the Tooltip element’s portal into. The default is `() => document.body` 
	 */
	parentSelector: PropTypes.func,
``
